### PR TITLE
Implement Blob url support for worker

### PR DESCRIPTION
**Pull request by kevinkassimo**
Related to #2726 

Support worker creation using Blob URL.

Notice that it does not support static import statements (It seems to me that Chrome also does not support it with both blob url and data url?)

(Data URL version is left to implement later since there are more complications involved)

### CODE REVIEW

File: js/url.ts [hunk 1]
```diff
@@ -63,6 +66,20 @@ function parse(url: string): URLParts | undefined {
   return undefined;
 }
 
+// Based on https://github.com/kelektiv/node-uuid
+// TODO(kevinkassimo): Use deno_std version once possible.
```
Comment (ry) [inline js/url.ts]: 👍 
I think I now have an ideas now on how to use deno_std well (#2711)

File: tests/040_worker_blob.ts.out [hunk 2]
```diff
@@ -0,0 +1 @@
+code from Blob
```
Comment (ry) [inline tests/040_worker_blob.ts.out]: cool.

**Comment by qwerasd205** [comment]
I figured I'd go ahead and reference #2797 here since this is where the feature with the bug in it was created. (summary of #2797: Workers created from blobs cause panic when error is thrown). If I shouldn't bring up issues on merged PRs please let me know for future reference.

**Comment by EliteScientist** [comment]
Was this reverted in 1.4.1? I'm having issues with my project because of this:

- fix(cli/js): disable URL.createObjectUrl (#7543)

**Pull request merged**